### ISSUE TITLE
`fit!` not exported in 0.19.3/0.19.4?

### ISSUE THREAD

**Issue by jbrea** [alan-turing-institute/MLJ.jl#1041]
I just updated to `0.19.4` (same for `0.19.3`) and now I don't see `fit!` exported anymore.
```julia
julia> using MLJ

julia> fit!
ERROR: UndefVarError: `fit!` not defined

(jl_2LAEwn) pkg> st
Status `/tmp/jl_2LAEwn/Project.toml`
⌃ [add582a8] MLJ v0.19.4
```
Is this intentional? With `0.19.2` `fit!` is exported.

**Comment by pebeto** [comment]
Solved in 70a4ed1. Waiting for a merge to master and a new release.

**Comment by pebeto** [comment]
Now live (JuliaRegistries/General#90360). Closing this issue.
